infra/build/developer-tools/build/scripts/task_helper_functions.sh

```diff
@@ -305,6 +305,17 @@ function generate_docs() {
   done < <(find_files . -name '*.tf' -print0 \
     | compat_xargs -0 -n1 dirname \
     | sort -u)
+  
+  echo "Generating blueprint metadata"
```

Comment:
I am thinking we should temporarily guard this with an env var so we can control the initial rollout. This is also used within `check_documentation` so linter will fail for any repo tracking dev tools major.

Reply:
Done. I also excluded metadata.yaml in the diff until we solve the real problem.